packages/core/test.setup.cjs

```diff
@@ -1,6 +1,7 @@
 /* globals module require globalThis */
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const { MemoryStorage } = require('@medplum/core');
+require('web-streams-polyfill');
```

Comment:
Oh interesting, I guess this is necessary because `jsdom` doesn't have the WebStream API supported, right? I didn't even realize that

Reply:
yup it was seemingly unavailable 🤷